.github/workflows/e2e.yml

```diff
@@ -124,12 +124,12 @@ jobs:
             class: org.apache.streampark.e2e.cases.TeamManagementTest
           - name: MemberManagementTest
             class: org.apache.streampark.e2e.cases.MemberManagementTest
-          - name: ApplicationsFlink116OnYarnWithFlinkSQLTest
-            class: org.apache.streampark.e2e.cases.ApplicationsFlink116OnYarnWithFlinkSQLTest
-          - name: ApplicationsFlink117OnYarnWithFlinkSQLTest
-            class: org.apache.streampark.e2e.cases.ApplicationsFlink117OnYarnWithFlinkSQLTest
-          - name: ApplicationsFlink118OnYarnWithFlinkSQLTest
-            class: org.apache.streampark.e2e.cases.ApplicationsFlink118OnYarnWithFlinkSQLTest
+          - name: Flink116OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink116OnYarnTest
+          - name: Flink117OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink117OnYarnTest
+          - name: Flink118OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink118OnYarnTest
```

Comment:
Please avoid this unnessnary change.

Reply:
> Please avoid this unnessnary change.

This test case includes both Flink on YARN application and YARN per-job modes, so I think the name "ApplicationsFlink116OnYarnWithFlinkSQLTest" is too long and inaccurate. How about "FlinkSQL116OnYarnTest"?

Reply:
I'm ok with it.